packages/common/test-utils.js

```diff
@@ -200,3 +204,88 @@ function jlog(object) {
   console.log(JSON.stringify(object, null, 2));
 }
 exports.jlog = jlog;
+
+// This class runs a simulated Earthdata Login server
+class FakeEarthdataLoginServer extends http.Server {
```

Comment:
Nitpicky, but feels like this should be in its own file

Reply:
I buy that, especially since this could grow in size.

Reply:
Done.